### ISSUE TITLE
Warning message on startup

### ISSUE THREAD

**Issue by Arcanemagus** [atom/teletype#202]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Followed all applicable steps in the [debugging guide][debugging-guide]
    * Checked the FAQs on the [message board][message-board] for common solutions
    * Checked that your issue [isn't already filed][already-filed]?

[debugging-guide]: http://flight-manual.atom.io/hacking-atom/sections/debugging/
[message-board]: https://discuss.atom.io/c/faq
[already-filed]: https://github.com/atom/teletype/issues

### Description

When launching Atom a warning message is printed to the console related to `pusher.com`.

### Steps to Reproduce

1. Install `teletype`
2. Activate the package

**Expected behavior:**

No warning messages printed to the console.

**Actual behavior:**

The following message is printed:
```
Pusher : You should always specify a cluster when connecting. See: https://pusher.com/docs/javascript_quick_start
warn @ C:\Users\username\.atom\packages\teletype\node_modules\pusher-js\dist\node\pusher.js:1184
```

**Reproduces how often:**

100% of the time.

### Versions

OS: Windows 10 x64
`teletype@0.1.2`
```
Atom    : 1.24.0-dev-b2ec0067a
Electron: 1.6.15
Chrome  : 56.0.2924.87
Node    : 7.4.0
```

### Additional Information

Somewhat related to these dependencies only needing to be loaded when used, see #201.

**Comment by jasonrudolph** [comment]
It looks like https://github.com/atom/teletype/pull/204 might resolve this issue.

**Comment by as-cii** [comment]
The fix for the issue has just been published on teletype v0.1.3. Thanks for the report!
